devops/consuming_conan_center.rst

```diff
@@ -0,0 +1,71 @@
+  - If you are creating your own packages for your own libraries or applications, or any of the above reasons is a concern for your production environment, then the recommended approach is **hosting your own copy of the packages you need for production** in your own private server.
+
+We have seen many Conan users succesfully using the second approach, building and hosting their own binaries in their own repository, even for "consuming" cases. The process for doing that is not complicated (and we are working to further streamline it) and in practice it is advangeous in most cases. Let's see how this can be done:
+
+
+
+Creating and hosting your own ConanCenter binaries
+--------------------------------------------------
+
+Hosting your own copy of the packages you need in your server could be done by just downloading binaries from ConanCenter and then uploading them to your own server. However, it is much better to fully own the complete supply chain and create the binaries in your own CI systems. So the recommended flow to use ConanCenter packages in production would be:
+
+- Create a fork of the ConanCenter Github repository: https://github.com/conan-io/conan-center-index
+- Create a list of the packages and versions you need for your projects. This list can be added to the fork too, and maintained there (packages can be added and removed with PRs when the teams need them).
+- Create a script that first ``conan export`` all the packages in your list, then ``conan create --build=missing`` them. Do not add ``user/channel`` to these packages, it is way simpler to use them as ``zlib/1.2.13`` without user-channel. The ``user/channel`` part would be mostly recommended for your own proprietary packages, but not for open source ConanCenter packages.
+- Upload your build packages to your own server, that you use in production, instead of ConanCenter.
+
+This is the basic flow idea. We will be adding examples and tools to further automate this flow as soon as possible.
+
+
+This flow is relatively straightforward, and has many advantages that mitigate the above risks:
+
+- No central repository outage can affect your builds.
+- No changes in the central repository can break your projects, you are in full control when and how those changes are updated in your packages (as explained below).
+- You can customize, adapt, fix and perfectly control what versions are used, and release fixes in minutes, not weeks. You can apply customizations that wouldn't be accepted in the central repository.
+- You fully control the binaries supply chain, from the source (recipes) to the binaries, eliminating in practice the majority of potential supply chain attacks of central repositories.
```

Comment:
The second half of the sentence was a bit confusing.

Hopefully this captures the same idea

```suggestion
- You control the binaries supply chain, from the recipes to the binaries, eliminating in practice the majority of potential supply chain attacks seen in central repositories.
```